python/llm/src/ipex_llm/serving/api/model_worker.py

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import torch
+from transformers.utils import logging
+import time
+import asyncio
+from transformers import TextIteratorStreamer
+logger = logging.get_logger(__name__)
+
+
+class ModelWorker:
```

Comment:
Rename `ModelWorker` and `ModelRunner` to `ModelWorker` and `PPModelWorker`？The current name may be kind of confusing